corehq/apps/integration/util.py

```diff
@@ -9,5 +11,30 @@ def domain_uses_dialer(domain):
         return False
 
 
+def domain_uses_hmac_callout(domain):
+    try:
+        settings = HmacCalloutSettings.objects.get(domain=domain)
+        return settings.is_enabled
+    except HmacCalloutSettings.DoesNotExist:
+        return False
+
+
 def get_dialer_settings(domain):
     return DialerSettings.objects.get(domain=domain)
+
+
+def integration_contexts(domain):
+    context = {'dialer_enabled': domain_uses_dialer(domain)}
+    if domain_uses_hmac_callout(domain):
+        settings = HmacCalloutSettings.objects.get(domain=domain)
```

Comment:
instead of doing the `get` twice how about this:
```python
def get_hmac_callout_settings(domain):
    try:
        settings = HmacCalloutSettings.objects.get(domain=domain)
        return settings if settings.is_enabled else None
    except HmacCalloutSettings.DoesNotExist:
        pass

settings = get_hmac_callout_settings(domain)
if settings:
    ...
```